meg_runtime/ui/pluginspanel.py

```diff
@@ -0,0 +1,100 @@
+
+from PyQt5 import QtWidgets
+
+from meg_runtime.plugins import PluginManager
+from meg_runtime.ui.basepanel import BasePanel
+from meg_runtime.ui.manager import UIManager
+
+class PluginsPanel(BasePanel):
+    """Setup the plugin panel."""
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def get_title(self):
+        """Get the title of this panel."""
+        return 'Plugins'
+
+    def on_load(self):
+        """Load dynamic elements within the panel."""
+
+        # Attach handlers
+        instance = self.get_widgets()
+
+        self.enable_button = instance.findChild(QtWidgets.QPushButton, 'enableButton')
+        self.enable_button.clicked.connect(self.enableCurrentPlugin)
+        self.disable_button = instance.findChild(QtWidgets.QPushButton, 'disableButton')
+        self.disable_button.clicked.connect(self.disableCurrentPlugin)
+        self.uninstall_button = instance.findChild(QtWidgets.QPushButton, 'uninstallButton')
+        self.uninstall_button.clicked.connect(self.uninstallCurrentPlugin)
+        self.add_button = instance.findChild(QtWidgets.QPushButton, 'addButton')
+        self.add_button.clicked.connect(UIManager.open_add_plugin)
+
+        self.plugin_list = instance.findChild(QtWidgets.QTreeWidget, 'pluginList')
+        self.plugin_list.itemSelectionChanged.connect(self.changeButtonStates)
+
+        self.refreshPluginList()
+
+    def refreshPluginList(self):
+        self.plugin_list.clear()
+        plugins = PluginManager.get_all()
+        for plugin in plugins:
+            self.plugin_list.addTopLevelItem(QtWidgets.QTreeWidgetItem([
+                chr(128309) if plugin.enabled() else chr(9898),
+                plugin.name(),
+                plugin.version(),
+                plugin.author(),
+                plugin.description()
+            ]))
+        
+        # disable buttons
+        self.changeButtonStates()

```

Comment:
Same as the other panel, would benefit from the other method of handling the panel interactions.